sdk/metric/internal/aggregate/sum_test.go

```diff
@@ -208,8 +202,8 @@ func TestPreComputedDeltaSum(t *testing.T) {
 	agg.(precomputeAggregator[int64]).aggregateFiltered(3, attrs)
 	agg.(precomputeAggregator[int64]).aggregateFiltered(10, attrs)
 	got = agg.Aggregation()
-	// measured(+): 5, previous(-): 1, filtered(+): 13
-	want.DataPoints = []metricdata.DataPoint[int64]{point[int64](attrs, 17)}
+	// measured(+): 5, previous(-): 8, filtered(+): 13
+	want.DataPoints = []metricdata.DataPoint[int64]{point[int64](attrs, 18)}
```

Comment:
`5-8+13` is `10`, why do we expect `18` here?

Reply:
Forgot to update the comment.  The previous should be 0, since the attribute set was not observed in the previous collection cycle.